### ISSUE TITLE
Allow for the exclusion of cookies and by prefix to allow for dynamic ranges

### ISSUE THREAD

**Issue by marcrobertscamao** [Flowpack/Flowpack.FullPageCache#20]
In many cases specifically related to affiliate links there are dynamic cookies and dynamic query parameters generated in order to support this case I would recommend that it is possible to exclude entire ranges of parameters based on prefix, such as the following range of cookies where `<wpid>` is set from the partner domain `_ga_<wpid>`

My suggestion would be to configure prefixes as follows:

```yaml
Flowpack:
  FullPageCache:
    request:
      cookieParams:
        ignorePrefix:
          - _ga_
      queryParams:
        ignorePrefix:
          - utm_
```

**Comment by marcrobertscamao** [comment]
I will open up a merge request with a first draft so that we can review.

**Comment by marcrobertscamao** [comment]
Closing as resolved through #22
